auth_brute_force/README.rst

```diff
@@ -48,14 +37,14 @@ Logging
 This module generates some WARNING logs, in the three following cases:
 
 * Authentication failed from remote '127.0.0.1'. Login tried : 'admin'.
-  Attempt 1 / 10.
+Attempt 1 / 10.
```

Comment:
Why do you remove indentation? This provokes an incorrect RST syntax. It was correct as it was.